beeai_framework_starter/helpers/io.py

```diff
@@ -24,11 +26,15 @@ def __next__(self) -> str:
         try:
             while True:
                 prompt = input(colored(self.input, "cyan", attrs=["bold"])).strip()
+                if not sys.stdin.isatty():
+                    print(prompt)
 
                 if prompt == "q":
                     raise StopIteration
 
-                prompt = prompt if prompt else self.fallback
+                if not prompt:
+                    prompt = self.fallback
+                    self.write(self.input, prompt)
```

Comment:
This is a nice add, we should include in the main repo version as well. One small nit that I'm on the fence about, since this uses self.write it looks like the following, but I think it's still better than just print which will still print on a new line 
![Screenshot 2025-04-14 at 2 49 59 PM](https://github.com/user-attachments/assets/ea1b855b-5bb2-485a-b76d-0a1bb13773d5)

Reply:
Exactly.